.github/workflows/pr-test.yaml

```diff
@@ -18,3 +25,48 @@ jobs:
         run: bundle install
       - name: Run tests
         run: bundle exec rake unit_test
+
+  task-helm-acceptance:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: nolar/setup-k3d-k3s@v1
+        with:
+          version: v1.19
+          k3d-args: -p 80:80@loadbalancer -p 443:443@loadbalancer
+      - uses: ./.github/actions/install-bolt
+      - uses: ./.github/actions/install-k8s-tools
+      - name: Install test helm chart
+        run: |-
+          bolt task run pam_tools::helm_install_chart --targets=localhost chart=./.github/test-chart release='test-release' values='name: testing'
+      - name: Check configmap
+        run: |
+          kubectl get configmap/test-chart-configmap -o yaml | grep 'Hello testing'
+      - name: Delete resources
+        run: |
+          bolt task run pam_tools::delete_k8s_app_resources --targets=localhost kots_slug=test-chart
+
+  plan-install-acceptance:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Prepare license
+        run: printenv CONNECT_LICENSE > license.yaml
+      - uses: nolar/setup-k3d-k3s@v1
+        with:
+          version: v1.19
+          k3d-args: -p 80:80@loadbalancer -p 443:443@loadbalancer
+      - uses: ./.github/actions/install-bolt
+      - uses: ./.github/actions/install-k8s-tools
+      - name: Run install_published plan
+        run: |
+          bolt plan run pam_tools::install_published targets=localhost license_file="$(pwd)/license.yaml" app_timeout=900 kots_install_options="--skip-preflights=true"
```

Comment:
Not sure this will work with defaults. It only has 2 CPUs; we do some munging in https://github.com/puppetlabs/holodeck-manifests/blob/main/Makefile#L666-L678 to make that fit.

Reply:
Yeah, I was a little surprised that this has been working, actually. The pam_tools::kurl_install still has the default config tuning that kurl_test needs to fit the kurl vm, so maybe that's been sufficient. Or, actually, the k3d services may just not be trying to enforce limits? Isn't that implementation dependent?

Anyway, I'm planning on pulling out the current tight default config, so that kurl_test supplies it when it calls install_published. May do that work next and see what falls out.

Reply:
Yep, the default config in kots_install was letting it squeak by. That's adjustable somewhat now, and the workflow sets the memory for the plan to match the runner env.